src/sentry/south_migrations/0455_auto__add_field_groupenvironment_first_seen.py

```diff
@@ -0,0 +1,1259 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+from sentry.utils.db import is_postgres
+
+
+class Migration(SchemaMigration):
+
+    # Flag to indicate if this migration is too risky
+    # to run online and needs to be coordinated for offline
+    is_dangerous = True
+
+    def forwards(self, orm):
+        # Adding field 'GroupEnvironment.first_seen'
+        if is_postgres():
+            db.commit_transaction()
+            db.execute('ALTER TABLE sentry_groupenvironment ADD COLUMN first_seen timestamp with time zone')
```

Comment:
Don't you need `DEFAULT NULL` somewhere in here? Or is that implied? I forget.

Reply:
seems like it defaults to null based on local testing and https://www.postgresql.org/docs/9.3/ddl-default.html

> If no default value is declared explicitly, the default value is the null value.

Reply:
🥇